content/try-matrix.md

```diff
@@ -2,4 +2,8 @@
 title = "Try Matrix"
 template = "try-matrix.html"
 extra.summary = "Join the Matrix ecosystem!"
+extra.meta_description = """
+To try Matrix, pick a client, pick a server (or use matrix.org if you're not
```

Comment:
I feel like this is weird to have "(or use matrix.org if you're not sure)" like this in here. It makes sense  in the page itself but is weird without context in google